cdap-data-fabric/src/main/java/co/cask/cdap/data2/util/hbase/TableId.java

```diff
@@ -76,6 +92,45 @@ public static TableId from(String tablePrefix, String namespace, String tableNam
     return new TableId(tablePrefix, Id.Namespace.from(namespace), tableName);
   }
 
+  public static TableId from(String hBaseNamespace, String hTableName) {
```

Comment:
This logic should be in this class. They should go into corresponding `HTableNameConverter`